### PR TITLE
[olm] operator metrics e2e periodic job

### PR DESCRIPTION
**Pull request by Bowenislandsong**
OLM would like to have a daily cron job to test that the red hat operators with metrics are behaving.

/cc @ecordell 

/cc @ravisantoshgudimetla  Could you also give a quick look thanks!

### CODE REVIEW

**Comment by Bowenislandsong** [comment]
/hold
Hold till the test for olm is implemented. Please review so that things can go in parallel.

**Comment by Bowenislandsong** [comment]
/cc @shawn-hurley 
Please let me know if you have any suggestions about the naming of the test.

File: ci-operator/jobs/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      - name: TEST_COMMAND
+        value: make e2e-operator-metrics
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /usr/local/e2e-aws-olm-cluster-profile
```
Comment (ecordell) [inline ci-operator/jobs/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master-periodics.yaml]: A little weird that these are named after the other job but afaik that should be fine

**Pull request merged**